qt/about.cpp

```diff
@@ -15,7 +15,7 @@
 #include <QtWidgets/QVBoxLayout>
 
 AboutDialog::AboutDialog(QWidget * parent)
-  : QDialog(parent, Qt::WindowTitleHint | Qt::WindowSystemMenuHint)
+  : QDialog(parent, Qt::WindowTitleHint | Qt::WindowSystemMenuHint | Qt::WindowCloseButtonHint)
```

Comment:
```suggestion
  : QDialog(parent)
```
It looks good without any additional flags.
![image](https://github.com/user-attachments/assets/0a1d5f72-1cb9-4905-8166-514a4c08244b)

Reply:
> Why employ html when it risks over-engineering a solution? Could not a humble widget suffice, elegantly meeting our needs without undue complexity?

I agree, we can make the app display HTML with JavaScript using QWebEngineView, but for that we would need to use QtWebEngine, which doesn't seem to be used in a project anywhere currently? So it's an additional dependency.

Reply:
Added explicit flags for "Maximize window" and "Close window", I think this should be enough.